### PR TITLE
Add commit to master

### PR DESCRIPTION
**Pull request by munozemilio**

### CODE REVIEW

File: build/botframework-cli-mac.yml [hunk 1]
```diff
@@ -28,7 +28,7 @@ pr:
 jobs:
   - job: CLI
     variables:
-      buildVersion: '4.10.0-preview.$(Build.BuildId)'
+      buildVersion: '4.9.0-RC1'
```
Comment (cleemullins) [inline build/botframework-cli-mac.yml:31]: These are incorrect. Master should say as the 4.10.0-preview XYZ build.

File: build/botframework-cli.yml [hunk 2]
```diff
@@ -28,7 +28,7 @@ pr:
 jobs:
   - job: CLI
     variables:
-      buildVersion: '4.10.0-preview.$(Build.BuildId)'
+      buildVersion: '4.9.0-RC1'
```
Comment (cleemullins) [inline build/botframework-cli.yml:31]: These are incorrect. Master should say as the 4.10.0-preview XYZ build.

File: packages/dialog/src/library/schemaMerger.ts [hunk 3]
```diff
@@ -295,7 +296,78 @@ export default class SchemaMerger {
         return ppath.normalize(path)
     }
 
-    // Expand .csproj packages and projects
+    // Expand nuget package and all of its dependencies
+    async expandNuget(packageName: string, minVersion: string, root: boolean): Promise<string[]> {
```
Comment (cleemullins) [inline packages/dialog/src/library/schemaMerger.ts:300]: This is a lot of new complex code, with no tests. 

Let's get tests added to this PR before we merge it.

**Comment by munozemilio** [comment]
I just created a clean PR with a cherry-pick

**Pull request closed without merging**